src/stt/include/stt/SteinerTreeBuilder.h

```diff
@@ -100,6 +100,12 @@ class SteinerTreeBuilder
                        const std::vector<int>& y,
                        const std::vector<int>& s,
                        int acc);
+
+  // run FOARS
+  Tree makeFoarsTree(const std::vector<int>& x_pin,
```

Comment:
Did you intend to include this in this PR?  It sems unrelated.

Reply:
not sure why this happened again ... removed foars content in latest PR